### ISSUE TITLE
Screen readers issues. 

### ISSUE THREAD

**Issue by kkrawczyk123** [benetech/MathShare#378]
This are the results of comparison testing of VoiceOver, ChromeVox, NVDA and JAWS screen readers before and after react transition on different browsers. 
- VoiceOver is sensitive for html elements even on a version without react implementation. I tested that screen reader on Safari and Chrome both. Alerts are not read at all on non-react version, however, I think that we have regression on the ability to read alerts on react version too so it would require fixing.

- I tried both versions with ChromeVox on MacOS and Windows operating systems. My impression is much better with that screen reader. I didn’t hear any html elements which is good. Both react and non-react versions didn’t differentiate that much. All alerts were visible for screen reader on react version. I think we have the best fit with that screen reader right now. 

- I verified NVDA with chrome, firefox, and edge browsers. I can’t say our app is adapted to that screen reader. The best result I obtained on chrome browser - some html accents were noticeable but alerts were visible and read correctly. I can’t say the same about Firefox and edge as results for them are comparably bad. In that case, html elements are not such a big deal as NVDA reads everything that is on problem page - all shortcut. Alerts are not readable at all for both browsers too. Testing of the non-react version on Edge was additionally difficult as there were visible a lot of bugs which are already fixed on react version. In general, NVDA has huge problems with reading mathematical expressions, even numbers and symbols are problematic.  

- JAWS is the most sensitive for html elements, especially on the Chrome browser. For both react and non-react version whole content of the main page is read. On the react version warnings are visible for JAWS but there are also a lot of randoms and repeats which occur with all screen readers but this behavior is kind of random and not reproducible. It is hard to evaluate html sensitiveness on the Firefox browser as it acts random, once is read, second is not. In general, there is a problem with reading react warnings on firefox browser.

**Comment by johnhbenetech** [comment]
@NSoiffer I would love for you to take a look at our development version for opinions about what our strategy should be for screen reader support.

What are the known "good" combinations of browser and screen readers that we know we've had success with? Are there any easy modifications we can make to help with structuring the page?

I found this: https://github.com/benetech/MathShare/wiki/Using-Accessibility
But im not sure if that was actually done in the previous static version
